### DISCUSSION TITLE
Vector of reactors?

### DISCUSSION THREAD

**Discussion by stustd** [lf-lang/lingua-franca#1057] [General]
Instead of using (unique) variable names, is there a way to create a number of reactors in a set? For example, in a `main reactor` this is what I'm trying to achieve (which obviously, as I know, has no bearing in official LF, but illustrates the point):
```
...
f = new foo();
std::vector<bah> b;
for (int i=0; i<10; i++) {
    b.add(new bah());
    f.y -> b.back().x;
}
...
```
The advantage would be in the logic to be able to refer to a reactor by its index instead of using `switch` statements.

**Comment by edwardalee** [comment]
See https://github.com/lf-lang/lingua-franca/wiki/Multiports-and-Banks-of-Reactors#banks-of-reactors
